---
title: "Evidence for Paraphyletic Phylogeny of Monogenea using modern NGS Data and Multigene Analysis"
subtitle: 'University of Graz, Institut for Biology, AMEB2020'
author: "Hannes Oberreiter, Anna Dünser"
date: "July-August, 2020"
header-includes:
    - \usepackage{hyperref}
    - \usepackage{float}
    - \usepackage{caption}
    - \captionsetup[figure]{font=small}
output:
  bookdown::pdf_document2:
    extra_dependencies: ["float"]
    toc: false
    citation_package: biblatex
bibliography: bib.bibtex
biblio-style: authoryear
link-citations: true
urlcolor: blue
abstract: Using publicly available genomic data and additionally data, which we produced ourselfs using modern NGS technology, we try to resolve the systematic position in the neodermata. A supermatrix of 313 BUSO genes were used with raxml to build a phylogenetic tree. Result shows Polyopisthocotylean (Monogenea) and Trematoda as monophyletic group with sister taxa the Cestodes. Monopisthocotyleans (Monogenea) can be seen as basal group inside the Neodermata. This new insight could open discussion if the life-style (ecto- vs. endoparasitic) and life history (simple vs. complex) could have evolved two times in the Neodermata group. Close relationship between Polyopisthocotylean (Monogenea) and Trematoda could explain similar feeding habits between those two taxa.
---
```{r global_options, include=FALSE}
# this tells r to fix images on positions were I want them and don't let LaTeX shuffle them around like crazy
knitr::opts_chunk$set(fig.pos = "H", out.extra = "", fig.align="center")
library("tidyverse")
```

# Introduction

Neodermata are obligat parasitic plathelminthes, subsuming three groups of flatworms including the Trematoda (flukes), Cestoda (tapeworms) and the Monogenea [@littlewood1999; @park2007; @gibson2014]. Most Trematodes and Cestodes are endoparasitic with a complex life cycle, while Monogeneans are mainly adjusted to a ectoparasitic lifestyle with a direct life cycle [@gibson2014]. The Monogeneans, which mostly parasite aquatic vertebrates, can be further subdivided into the Polyopisthocotyleans, that possess more complex haptors, with clamps and minuscule hooks, and are comprised mostly of blood-feeders that attach to the gills of their hosts. The second subsclass are the Monopisthocotyleans, which possess more simple haptors, only hooks, and can be found on skin or gills and feed on skin and or muscus [@gibson2014].

The evolutionary history of these three groups, as well as the origin of their differing life cycles (complex vs. simple) and the origin of the differing parasitic lifestyles (endo-/ectoparasitism) have been and are extensively debated. To gain more insight into the evolutionary history of the neodermata the phylogenetic position of the Monogeneans has to be resolved [@park2007].  At the moment there are multiple competing hypotheses about the correct evolutionary history of the neodermata [@littlewood1999; @olson2001; @lockyer2003; @perkins2010; @laumer2014]. 

A special species in our group was the monogenean *Dictyocotyle coeliaca*, which belongs to the subsclass of Monopisthocotyleans. The taxa is quite rare and morphologically specialized [@dawes1948; @dawes1958]. It can be found in the cœlom of its hosts where it was found multiple times on liver tissue [@dawes1948]. Possible hosts include a variety of deep sea rays, like *Raja radiata* and *R. lintea* [@dawes1948]. *D. coeliaca* belongs to the subclass of Monopisthocotyleans.

Our goal is with the addition of newly produced sequencing data of two species of monogeneans (*D. coeliaca* and *Leptocotyle minor*), the aim of this study is to further improve already existing sequencing data and, with the addition of openly accessible data from NCBI Genbank and a Multigene Matrix shed more light onto the controversely discussed phylogeny of the neodermata with a focus on the positioning of the monogenenans.

# Material and Methods

## DNA extraction and library prep

The specimen of *D. coeliaca* were sampled in Norway from the host species *R. radiata*. The Zymo Research DNA Miniprep Plus KIT was used to extract DNA. After the extraction the concentration and length distribution were assessed with a Qubit 4 Flourometer using the broad range assay resulting in an concentration of 69.6ng/µl. The TapeStation 2200 (Agilent Technologies) using genomic DNA screen tapes was used to assess size distribution, see fig. \@ref(fig:tapestation). To reach the recommended concentration of 1µg of high molecular weight DNA two sample specimen (20µl B164-65, 21µl B161-63) were pooled for the library preparation. Library preparation was done using the Nanopore Ligation Sequencing Kit (SQK-LSK109). To enrich sequences with a length of 3kb or longer, we used the long frame buffer following the protocol (Lab Protocol Version `GDE_9063_v109_revT_14Aug2019` which can be found on a github repository: [AMEB 2020 practical](https://github.com/chrishah/AMEB_2020_practical)).

The concentration of the prepared library was assessed using a Qubit 4 Flourometer. Quality assessment in library preparation followed the lab manual. The first Qubit measurement for *D. coeliaca* did result in 12.4ng/µl (total 744ng) and the second in 47.2ng/µl (total 660.8ng). For sequencing we used the minION platform of the Oxford Nanopore Technologies (ONT). The loading concentration of DNA in our samples was 566.4ng. Additionally DNA sequencing was also conducted by a second group of the practical course for the second monogenen species, *L. minor*.

(ref:tape-label) TapeStation, Analysis Software A.02.01 SR1, report for *D. coeliaca*

```{r tapestation, fig.cap="(ref:tape-label)", echo=FALSE, fig.width = 2}
knitr::include_graphics("data/tape-station.png")
```

## Public Gene Data

Phylogenetic tree analysis was done with public available data from [NCBI Genbank](https://www.ncbi.nlm.nih.gov) and local available assemblies from the University of Graz. The full list of sources can be found in table \@ref(tab:samples). Some additional outgroup taxa were added, which are not shown in the table. 


```{r samples, echo=FALSE}
t <- read.delim("data/public.tsv")
t$species <- paste("*", t$species, "*", sep="")
t$description <- str_replace_all(t$description, "https://www.ncbi.nlm.nih.gov/", "")
t$description <- str_replace_all(t$description, "-", "local sample")
t <- t[order(t$group, t$species),]

n <- c("Taxa", "Group", "Source (https://www.ncbi.nlm.nih.gov/*)")
knitr::kable(
  t[,1:3], booktabs = TRUE, row.names = FALSE,
  caption = 'List of taxa and corresponding phylogenetic group used in the analysis. The "Turbellaria" taxa was used as outgroup. Source links are public available record in the ncbi databank.',
  col.names = n, format = "markdown"
)

```

## Genome Assembly

Most of the utilized software was stored in docker containers. The genome assembly and tree caclulation was done on a computer cluster, steps are reproducible because we facilitated docker^[https://www.docker.com/] container with singularity [@kurtzer2017].

In addition to the minION reads we also used local available Illumina reads of high quality. Illumina reads were quality corrected with TRIMMOMATIC^[docker://chrishah/trimmomatic-docker:0.38], for settings see code definition \@ref(def:code) [@bolger2014].

```{definition, label="code", name="Trimmomatic"}
Settings for Trimmomatic used for quality control of Illumina data.

~~~bash
trimmomatic PE -phred33
<input-output files>
ILLUMINACLIP:/usr/src/Trimmomatic/0.38/Trimmomatic-0.38/adapters/TruSeq3-PE-2.fa:2:30:10 \
LEADING:30 TRAILING:30 SLIDINGWINDOW:5:20 MINLEN:500
~~~
  
```

Assembly of contigs and scaffolds was done with PLATANUS^[docker://chrishah/platanus:v1.2.4], standard settings. Long reads from our ONT lab work were used in combination with the resulting scaffolds with pyScaf^[docker://chrishah/pyscaf-docker] logic to improve the quality of the assembly.

On all samples including our sequenced samples a BUSCO^[docker://chrishah/busco-docker:v3.1.0] analysis was done for gene labeling and quality control. The Metazoa `metazoa_odb9` lineage dataset was used with mode `genome`, self-training August optimization mode (`--long`), because our given non-model species, and `schistosoma` in the species select flag.

The phylogenetic tree building was archived with a `snakemake` workflow [@koster2012]. The workflow was generated by Christoph Hahn from the University of Graz and can be accessed via GitHub Repository^[https://github.com/chrishah/AMEB_HPC_Snakemake]. It uses a custom script to extract ortholog and paralog genes from BUSCO, sequence aligning with clustalW, post alignment rating and masking with ALISCORE and ALICUT and raxml for protein gene model finding of each gene and single tree building, post-filtering with ncbi-BLAST and FASconCAT to generate a multigene alignment tree, which can be used to create a supermatrix. The tree calculation using all genes inside the matrix was done with raxml. Full list of software and usage can be found in table \@ref(tab:software).

```{r software, echo=FALSE}
t <- read.csv("data/software.csv")
n <- c("Name", "Version", "Docker", "Application")
t <- t[order(t$program),]
knitr::kable(
  t[,c(1:3)], booktabs = TRUE, row.names = FALSE,
  caption = 'Full list of software and version. Docker representate the link to the docker ID, if "local" no docker container was used and the software was installed locally on the cluster server.',
  col.names = n[1:3]
)

```

## Visualization

Graphical representation of the results was done with R [@r2020] and the packages [@tidyverse2019; @treeio2020; @ggtree2020; @ape2019]. The script for visualization and text for this homework can be found on github^[https://github.com/HannesOberreiter/AMEB2020_dictyocotyle_coeliaca].

# Results

The quality analysis with BUSCO did show lower completeness for the local available Illumina assembly of *D. coeliaca*, which was improved with our ONP reads from the practical course. Because of the low length of fragments we did stop the minION sequencing early to save resources for other for the other groups in the course, the report for the minION sequencing is online and public available as PDF on github^[[minION Report](https://github.com/chrishah/AMEB_2020_practical/blob/master/assembly-long/MinION/sequencing_reports/minion_report_Dceol_010720.pdf)]. Therefore, we used the local assembly from Illumina sequences, without ONP reads, for further analysis. The BUSCO analysis for the assembly of *L. minor*, which only included the minION data and was done by another group in the course, did result in 87 BUSCO genes and was used for further analysis.

```{r genetree, fig.cap="Singel gene phylogenetic trees for genes available for all given taxa.", echo=FALSE, fig.width = 2}
knitr::include_graphics("data/gene_tree.pdf")
```

In total 313 BUSCO genes were used in the supermatrix to calculate the phylogenetic tree and four of these genes were available throughout all taxa, see fig. \@ref(fig:genetree). In none of the calculated single gene trees the Monogenean represent a monophyletic group. The resulting phylogenetic tree for the supermatrix with all BUSCO genes can be seen in fig. \@ref(fig:tree). It shows high bootstrap coverage for all nodes expect one internal clade node, but did not influence the phylogenetic position of the groups in question. 

The result for the parasitic groups indicates Monopisthocotylea (Monogenea) as the most basal clade and Cestoda as sister group to the monophyletic group of Trematoda and Polypisthocotylea (Monogenea). Therefore, in our results the major parasitic group of Monogenea (Monopisthocotylea and Polypisthocotylea) can be seen as paraphyletic. 

```{r tree, fig.cap="Phylogenetic tree of the major parasitic groups, clades with blue background belong to parasitic group of Monogenea.", echo=FALSE}
knitr::include_graphics("data/tree.pdf")
```

# Discussion

The cause why our minION reads did not improve the quality of the local available *D. coeliaca* Illumina assembly could not be identified but was probably a high fragmentation of the DNA while library preparation, as the DNA assessment after extraction did look good, see fig. \@ref(fig:tapestation). These resulted in very short and low coverage of these reads and no advantage for the Illumina assembly of was gained by it, if the ONP reads were used as alignment in scaffolding.

Our resulting tree does not correspond to the currently proposed hypotheses of the major parasitic groups, see figure \@ref(fig:trees). The phylogeny shows the group of Monogenea as paraphyletic, which is in accordance with new molecular based findings [@littlewood1999; @olson2001; @perkins2010; @laumer2014]. Previous studies which did group Monogenea in a monophylum, for example with morphological features alone or rRNA, are potentially wrong with their evolutionary history [@lockyer2003; boeger1993]. Our findings of the potential evolutionary history of the Neodermata are different from the previous studies and we want to try explain it, what this would mean for evolution of various traits in this group of big and diverse parasites. 


(ref:trees-label) Three competing hypotheses of the evolution of Neodermata previously found supported by molecular data. a) [@lockyer2003], b) [@perkins2010], and c) [@littlewood1999; @olson2001; @laumer2014]. Image from practical course worksheet available at [google.doc](https://docs.google.com/document/d/1h2fNYv-hyIJfBnAZd0uEgk6zkS6bFyiQgQISwl4Qfh8).
```{r trees, fig.cap="(ref:trees-label)", echo=FALSE}
knitr::include_graphics("data/trees.png")
```

The paraphyletic position of Monopisthocotylean as basal group and Polyopisthocotylean as sister group to Trematoda could indicate two separate evolutionary events for the lifestyle in Neodermata evolutionary history (ecto- vs. endoparasitism). As the Tubellaria, which are the sister group inside Platyhelminthes to the Nermatoda, are free living we would expect that ectoparasitic lifestyle is the logical plesiomorphic trait for Neodermata. @park2007 came to a similar conclusion that ectoparasitism arose first but only one taxa was representing the Monogenea group in the study (Polyopisthocotylean, *Microcotyle sebastis*). Another study with more Monopisthocotyleans but again only one Polyopisthocotylean taxa (*M. sebastis*) did also came to a similar conclusion, but the resulting tree did show the Polyopisthocotylean as sister group to the monophyletic Trematoda and Cestoda, see fig. \@ref(fig:tapestation)-b) [@perkins2010]. Our results indicate the same basal group but with Polyopisthocotylean as sister group to Tremadota, we could propose that there was, a second evolutionary event happening back from endoparasitism to ectoparasitims. Therefore, the ectoparasitisms trait, which is shared by most Monopisthocotyleans and Polyopisthocotyleans, could be homoplasy. Of course we cannot say this for sure and the basal trait in the ancestors for Polyopisthocotyleans was never been lost as previously proposed [@park2007; @perkins2010].

The same as for life-style can be said for the evolutionary history of the life-cycle (complex vs. simple). Both Monogenea groups follow mostly a simple life-cycle, whereas Tremadoa life-cycle is more complex, and most of the time molluscs are involved, and Cestoda with generally one or two intermediate hosts with eggs from intestinal living adults leaving the host via feces [@gibson2014]. The evolutionary history could again have two distinct events from simple to complex and back to simple or the simple life-cycle in Polyopisthocotyleans could be still the ancestral trait. 

The evolution of the feeding habit question is a little bit more difficult, but could good makes sense in our new proposed phylogenetic tree. Our basal group of Monopisthocotylean feeds mostly on skin and or mucus [@gibson2014]. This type of feeding could be potentially the ancestral plesiomorphic trait but it would make also sense if the ancestor already did feed on blood like most Polyopisthocotyleans, the more sophisticated haptor design could be an evolution over time [@park2007]. The split of the Cestoda taxa could happened earlier in history as previous thought [@littlewood1999; @olson200; @perkins2010; @laumer2014]. It is probably not the ancestral features as Cestoda absorb their nutrients through their surface layer and did loose traits for normal digestion, e.g. guts [@gibson2014], which is more or less a point of no return, at least in the given time frame. The close relationship between Trematodes and Polyopisthocotyleans could make sense as both diets are similar, mostly feeding on blood, and the evolutionary speciation could have happened by different location niches on the host (ecto- and endoparasitic).

Overall, our new proposed tree should give room for new discussion and shows the need for molecular studies on multiple taxa representing one group and the advantages in using a supermatrix of different genes. 

# Acknowledgments

Hahn, Christoph, Ph.D 

# References
