---
title: "Using Nanopore Technologies (ONT) minION technology to sequence <i>Dictyocotyle coeliaca</i> and align in phylogenetic"
subtitle: 'University of Graz, Institut for Biology, AMEB2020'
author: "Hannes Oberreiter, Anna Dünser"
date: "July-August, 2020"
output:
  bookdown::pdf_document2:
    toc: false
bibliography: bib.bibtex
abstract: We found nothing at all ;)
---

# Introduction

The Parasitic Flatworm *Dictyocotyle coeliaca* (Nybelin, 1941) is a very fascinating parasite due to its rarity and its special morphological features, distinguihing it from the common cloacal parasite *Calicotyle kroyeri* [@dawes1948; dawes1958]. (Isn't it the only monogenean inside the body cavity?) It is living in the cœlom of its host and was found multiple times on the liver of the host [@dawes1948].Possible hosts were the parasit was found are all deep sea rays for example *Raja radiata* and *Raja lintea* [@dawes1948].

Our species of interest belongs to the subclass of Monopisthocotylea. The correct systematic position is still under active discussion. At the moment there are multiple competing hypotheses about the correct evolutionary history of the neodermata, a sister-group of planthemites [@lockyer2003; perkins2010; littlewood1999; olson2001; laumer2014].

In an advanced evolutionary practice course at the University of Graz, we did sequence three different species with NGS Oxford Nanopore, which all belong into the same group. With the help of these new gained data and additional open access data from NCBI Genbank we want to shed some more light into the controverse phylogeny of neodermata.

# Material and Methods

## DNA extraction and library prep

The species samples were found on host *Raja radiata* in Norway. The DNA concentration after extracting and length distribution were assessed with a TapeStation using Genomic DNA screen tapes and a Qubit 4 Flourometer using the broad range assay resulting in an concentration of 69.6 ng/µl. To reach the recommended concentration of 1µg of high molecular weight DNA two sample specimen (20µl B164-65, 21µl B161-63) were pooled before the library prep. Library prep was done using the Nanopore Ligation Sequencing kit (SQK-LSK109). To enrich sequences that are 3kb long or longer, we used the long frame Buffer following the protocol. Lab Protocol Version `GDE_9063_v109_revT_14Aug2019` which can be found on a github repositroy which was used over the course in which we sequenced the parasite: [AMEB 2020 practical](https://github.com/chrishah/AMEB_2020_practical).
The concentration of the preped library was assessed using a Qubit 4 Flourometer. Quality Assesment in Library preperation was done according to the lab manual the first Qubit fluorometer measurement did result in 12.4ng/µl (total 744ng) and second 47.2ng/µl (total 660.8ng). For sequencing we used the Oxford Nanopore Technologies (ONT) minION Platform. The loading concentration of DNA in our samples was 566.4ng.

## Public Gene Data

Phylogenetic tree analysis was done with public available data from [NCBI Genbank](https://www.ncbi.nlm.nih.gov) and assemblies from the University of Graz, please refere to \@ref(tab:list-samples).

\begin{table}[]
\centering
\caption{List of taxa for phylogenetic tree building}
\label{tab:list-samples}
\begin{tabular}{@{}llll@{}}
\toprule
Taxa & \multicolumn{1}{c}{Group} & \multicolumn{1}{c}{Description} & \multicolumn{1}{c}{web/local} \\ \midrule
\textit{Schmidtea mediterranea} & 'Turbellaria' & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_002600895.1} & web \\
\textit{Macrostomum lignano} & 'Turbellaria' & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_002269645.1} & web \\
\textit{Taenia solium} & Cestoda & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_001870725.1} & web \\
\textit{Echinococcus granulosus} & Cestoda & \url{https://www.ncbi.nlm.nih.gov/assembly/GCF\_000524195.1} & web \\
\textit{Hymenolepis microstoma} & Cestoda & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_000469805.3} & web \\
\textit{Schistosoma mansoni} & Trematoda & \url{https://www.ncbi.nlm.nih.gov/assembly/GCF\_000237925.1} & web \\
\textit{Clonorchis sinensis} & Trematoda & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_003604175.1} & web \\
\textit{Opisthorchis viverrini} & Trematoda & \url{https://www.ncbi.nlm.nih.gov/assembly/GCF\_000715545.1} & web \\
\textit{Protopolystoma xenopodi} & Polyopisthocotylea & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_900617795.1/} & web \\
\textit{Gyrodactylus salaris} & Monopisthocotylea & \url{https://www.ncbi.nlm.nih.gov/assembly/GCA\_000715275.1} & web \\
\textit{Diclidophora denticulata} & Polyopisthocotylea & - & local \\
\textit{Eudiplozoon nipponicum} & Polyopisthocotylea & - & local \\ \bottomrule
\end{tabular}
\end{table}

## Genome Assembly

For reproductibility all of the used software were can be found in docker containers. Assembly and Tree Building was done on a cluster server with singularity.

In addition to the minION reads we also used local available Illumina reads of high quality. Illumina reads were quality corrected with TRIMMOMATIC^[docker://chrishah/trimmomatic-docker:0.38], settings see \@ref(exm:trim-settings)

```{example, trim-settings}
Settings for Trimmomatic used for quality control of Illumina data
```
```bash
trimmomatic PE -phred33
<input-output files>
ILLUMINACLIP:/usr/src/Trimmomatic/0.38/Trimmomatic-0.38/adapters/TruSeq3-PE-2.fa:2:30:10 \
LEADING:30 TRAILING:30 SLIDINGWINDOW:5:20 MINLEN:500
```

Assembly of contigs and scaffolds was done with PLATANUS^[docker://chrishah/platanus:v1.2.4], standard settings. Long reads from our ONT lab work were used in combination with the resulting scaffolds with pyScaf^[docker://chrishah/pyscaf-docker] logic to improve the quality of the assembly.

On all samples including our sequenced samples a BUSCO^[docker://chrishah/busco-docker:v3.1.0] analysis was done for gene labeling and quality control. The Metazoa `metazoa_odb9` lineage dataset was used with mode `genome` and flag for self-training August optimization mode (`--long`), because our given non-model species. TODO was ist die sp flag?


# Results

The quality analysis with BUSCO did show lower completness for the assembly which was improved with ONP reads. Therefore, we used only the given Illumina Sequences. In further analysis.


# Discussion




@article{dawes1948,
	Author = {DAWES, BEN},
	Da = {1948/04/01},
	Date-Added = {2020-06-30 13:45:27 +0200},
	Date-Modified = {2020-06-30 13:45:27 +0200},
	Doi = {10.1038/161642c0},
	Id = {DAWES1948},
	Isbn = {1476-4687},
	Journal = {Nature},
	Number = {4095},
	Pages = {642--643},
	Title = {A Rare C{\oe}lom-Dwelling Trematode},
	Ty = {JOUR},
	Volume = {161},
	Year = {1948},
}

@article{lockyer2003,
  title = {Utility of Complete Large and Small Subunit {{rRNA}} Genes in Resolving the Phylogeny of the {{Neodermata}} ({{Platyhelminthes}}): Implications and a Review of the Cercomer Theory: {{PHYLOGENY OF THE NEODERMATA}}},
  shorttitle = {Utility of Complete Large and Small Subunit {{rRNA}} Genes in Resolving the Phylogeny of the {{Neodermata}} ({{Platyhelminthes}})},
  author = {Lockyer, A. E. and Olson, P. D. and Littlewood, D. T. J.},
  date = {2003-01-31},
  journaltitle = {Biological Journal of the Linnean Society},
  volume = {78},
  pages = {155--171},
  issn = {00244066, 10958312},
  doi = {10.1046/j.1095-8312.2003.00141.x},
  langid = {english},
  number = {2}
}

@article{perkins2010,
  title = {Closing the Mitochondrial Circle on Paraphyly of the {{Monogenea}} ({{Platyhelminthes}}) Infers Evolution in the Diet of Parasitic Flatworms},
  author = {Perkins, Elizabeth M. and Donnellan, Steve C. and Bertozzi, Terry and Whittington, Ian D.},
  date = {2010-09},
  journaltitle = {International Journal for Parasitology},
  shortjournal = {International Journal for Parasitology},
  volume = {40},
  pages = {1237--1245},
  issn = {00207519},
  doi = {10.1016/j.ijpara.2010.02.017},
  langid = {english},
  number = {11}
}

@article{littlewood1999,
  title = {The Interrelationships of All Major Groups of {{Platyhelminthes}}: Phylogenetic Evidence from Morphology and Molecules},
  shorttitle = {The Interrelationships of All Major Groups of {{Platyhelminthes}}},
  author = {Littlewood, D. T. J. and Rohde, K. and Clough, K. A.},
  date = {1999-01},
  journaltitle = {Biological Journal of the Linnean Society},
  volume = {66},
  pages = {75--114},
  issn = {00244066},
  doi = {10.1111/j.1095-8312.1999.tb01918.x},
  langid = {english},
  number = {1}
}

@article{olson2001,
  title = {Interrelationships and {{Evolution}} of the {{Tapeworms}} ({{Platyhelminthes}}: {{Cestoda}})},
  shorttitle = {Interrelationships and {{Evolution}} of the {{Tapeworms}} ({{Platyhelminthes}}},
  author = {Olson, Peter D. and Littlewood, D.Timothy J. and Bray, Rodney A. and Mariaux, Jean},
  date = {2001-06},
  journaltitle = {Molecular Phylogenetics and Evolution},
  shortjournal = {Molecular Phylogenetics and Evolution},
  volume = {19},
  pages = {443--467},
  issn = {10557903},
  doi = {10.1006/mpev.2001.0930},
  langid = {english},
  number = {3}
}

@article{laumer2014,
  title = {Inclusive Taxon Sampling Suggests a Single, Stepwise Origin of Ectolecithality in {{Platyhelminthes}}: {{Ectolecithality}} in {{Platyhelminthes}}},
  shorttitle = {Inclusive Taxon Sampling Suggests a Single, Stepwise Origin of Ectolecithality in {{Platyhelminthes}}},
  author = {Laumer, Christopher E. and Giribet, Gonzalo},
  date = {2014-03},
  journaltitle = {Biological Journal of the Linnean Society},
  shortjournal = {Biol J Linn Soc Lond},
  volume = {111},
  pages = {570--588},
  issn = {00244066},
  doi = {10.1111/bij.12236},
  langid = {english},
  number = {3}
}


@article{dawes1958,
  title = {The {{Enigmatical Trematode}} “{{Dictyocotyle}} Coeliaca”},
  author = {Dawes, Ben and Griffiths, Ivor},
  date = {1958-10},
  journaltitle = {Nature},
  shortjournal = {Nature},
  volume = {182},
  pages = {1033--1034},
  issn = {0028-0836, 1476-4687},
  doi = {10.1038/1821033a0},
  langid = {english},
  number = {4641}
}






